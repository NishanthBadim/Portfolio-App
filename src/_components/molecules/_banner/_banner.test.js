import React from "react";
import { create } from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Banner from "./_banner";

describe("Banner testing", () => {
  test("Matches the snapshot", () => {
    const input = create(<Banner />);
    expect(input.toJSON()).toMatchSnapshot();
  })
})