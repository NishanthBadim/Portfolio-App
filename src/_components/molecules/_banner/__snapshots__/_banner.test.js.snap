// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Banner testing Matches the snapshot 1`] = `
<div
  className="makeStyles-bannerText-1"
>
  <h2
    className="makeStyles-h1-2"
  >
    Full Stack Web Developer
  </h2>
  <hr
    className="makeStyles-hr-3"
  />
  <p
    className="makeStyles-p-4"
  >
    HTML/CSS | Bootstrap | JavaScript | React | Spring Boot | MySQL
  </p>
  <div
    className="makeStyles-socialLinks-5"
  >
    <a
      href="https://in.linkedin.com/in/nishanth-badim-13326a177?trk=people-guest_people_search-card"
      rel="noopener noreferrer"
      target="_blank"
    >
      <div
        className="makeStyles-i-6"
      >
        <i
          aria-hidden="true"
          className="fa fa-linkedin-square"
        />
      </div>
    </a>
    <a
      href="https://github.com/Nishanth-17"
      rel="noopener noreferrer"
      target="_blank"
    >
      <div
        className="makeStyles-i-6"
      >
        <i
          aria-hidden="true"
          className="fa fa-github-square"
        />
      </div>
    </a>
    <a
      href="https://twitter.com/Nishanth_17"
      rel="noopener noreferrer"
      target="_blank"
    >
      <div
        className="makeStyles-i-6"
      >
        <i
          aria-hidden="true"
          className="fa fa-twitter"
        />
      </div>
    </a>
    <a
      href="https://www.instagram.com/_nishanth_17/?hl=en"
      rel="noopener noreferrer"
      target="_blank"
    >
      <div
        className="makeStyles-i-6"
      >
        <i
          aria-hidden="true"
          className="fa fa-instagram"
        />
      </div>
    </a>
  </div>
</div>
`;
